src/kit.ts

```diff
@@ -865,6 +865,19 @@ function convertMingwDirsToSearchPaths(mingwDirs: string[]): string[] {
   return mingwDirs.map(mingwDir => path.join(mingwDir, 'bin'));
 }
 
+export function configKitsPathForWorkspaceDirectoryPath(wsPath: string, kitFile: string): string {
+  return path.join(wsPath, kitFile);
+}
+
+export function configKitsPathForWorkspaceFolder(ws: vscode.WorkspaceFolder, kitFile: string): string {
+  return configKitsPathForWorkspaceDirectoryPath(ws.uri.fsPath, kitFile);
```

Comment:
Instead of hard coding this property to be relative to the workspace folder, we should enable variable substitution.  A quick and dirty way to support this is to import the `expandString` method from `expand.ts` and create a partial `ExpansionOptions` object to manage the substitutions.

```
const vars: expand.ExpansionVars = {
  workspaceRoot: ...,
  workspaceFolder: ...,
  buildType: "${buildType}",  // invalid substitution, just replace with itself
  workspaceRootFolderName: ...,
  generator: "${generator}", // invalid substitution
  userHome: ...,
  buildKit: "${buildKit}" // invalid substitution
};

return expand.expandString(kitFile, vars);

Reply:
Sounds like a nice idea. I did not see how to do this at first (as you need a driver for expansion options, which only exists after you have selected a kit). But I guess this should work. If you have a cleaner suggestion let me know as well!